### PR TITLE
commiting for steps 2 and 3, had some tech issues regarding tests. fi…

### PR DESCRIPTION
**Pull request by Daniel-J-94**
…xed by adding a return to the seed(testData) on line 9 of app.test.js, although it's possible this won't be a permenant fix

### CODE REVIEW

File: controllers/getCategoriesController.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const { getterCategories } = require("../models/getCategoriesModel");
+
+exports.getCategories = (req, res, next) => {
+  console.log("i'm in the grabCats controller!");
```
Comment (aloush-dev) [inline controllers/getCategoriesController.js]: make sure to remove console logs before submitted PRs

File: models/getCategoriesModel.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+const db = require("../db/connection");
+
+exports.getterCategories = () => {
+  console.log("I'm in the getCatModel!");
```
Comment (aloush-dev) [inline models/getCategoriesModel.js]: remove console logs

File: __tests__/app.test.js [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+});
+
+describe("Get all categories", () => {
+  it("200 should respond with an array of category objects, each of which should have a slug property and a description property", () => {
+    return request(app)
+      .get(`/api/categories`)
+      .expect(200)
+      .then((resultResponse) => {
+        const resultResponseBody = resultResponse.body;
+        console.log(
```
Comment (aloush-dev) [inline __tests__/app.test.js]: console logs

File: __tests__/app.test.js [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+          "I'm in the second test! Here is the resultResponseBody",
+          resultResponseBody
+        );
+        expect(resultResponseBody).toEqual({ message: "Doesn't exist" });
+      });
+  });
+  it("200 should respond with an array of category objects that has a length of 4 and 2 keys that are strings", () => {
```
Comment (aloush-dev) [inline __tests__/app.test.js:48]: Nice both your 200 tests are essentially testing the same thing, this one feels more robust as a standard to follow for the rest of your testing.

**Review by aloush-dev — APPROVED** [review]
Really nice PR.
Make sure next time to remove console logs before submitting.
Also the commit history would suggest you've been working on multiple tickets at a time.
Make sure to do the one you're curerntly on commit and PR and then move onto the next one on a new branch!

File: __tests__/app.test.js [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+          },
+          { slug: "dexterity", description: "Games involving physical skill" },
+          {
+            slug: "children's games",
+            description: "Games suitable for children",
+          },
+        ]);
+      });
+  });
+  it("404 should respond with an error message if path is spelled incorrectly", () => {
```
Comment (NRMSMITH) [inline __tests__/app.test.js:39]: This could be part of a different describe block if needed :)

File: controllers/getCategoriesController.js [hunk 6]
```diff
@@ -0,0 +1,11 @@
+const { getterCategories } = require("../models/getCategoriesModel");
+
+exports.getCategories = (req, res, next) => {
+  getterCategories()
```
Comment (NRMSMITH) [inline controllers/getCategoriesController.js]: change this to fetchCategories, even though it made me laugh ⭐

File: db/data/test-data/categories.js [hunk 7]
```diff
@@ -1,9 +1,9 @@
...
-    slug: 'social deduction',
-    description: "Players attempt to uncover each other's hidden role"
+    slug: "social deduction",
+    description: "Players attempt to uncover each other's hidden role",
   },
-  { slug: 'dexterity', description: 'Games involving physical skill' },
-  { slug: "children's games", description: 'Games suitable for children' }
+  { slug: "dexterity", description: "Games involving physical skill" },
+  { slug: "children's games", description: "Games suitable for children" },
 ];
```
Comment (NRMSMITH) [inline db/data/test-data/categories.js:9]: please avoid changing the data

**Review by NRMSMITH — APPROVED** [review]
Approved! 🚀 Please avoid changing the data and also make sure that you have named your functions in a stronger manner :)

**Pull request merged**